packages/panels/src/components/page-header/PageHeader.tsx

```diff
@@ -17,9 +17,14 @@ export const PageHeader: React.FC<PageHeaderProps> = ({
   children,
 }) => {
   return (
-    <Box shadow={"box"} background={cssColor("--lhds-color-ui-50")}>
+    <Box shadow={"box"} background={cssColor("--lhds-color-ui-50")} gap={2}>
       <Box indent={3}>
-        {renderBreadCrumbs && <Row spacing={1.25}>{renderBreadCrumbs()}</Row>}
+        {renderBreadCrumbs && (
+          <Row height={"32px"} alignItems={"center"}>
```

Comment:
Why height?

Reply:
That is the design.
The design was made with boxes with specific heights, so instead of trying to add spacing with correct height, I just added the height :)

Reply:
But we do support user-specified font-size that scales, right? The whole `rem`-business? Wouldn't it make more sense to use padding in that case?